src/core/sdl/window_sdl.cpp

```diff
@@ -198,6 +197,13 @@ bool WindowSDL::initSDL(uint32_t windowInitFlags)
     s_emscriptenFillBrowser();
 #endif
 
+#if RENDER_FULLSCREEN_ALWAYS // Use a full-screen on Android & PS Vita mode by default
```

Comment:
Looks like this comment needs to be extended: there are more platforms than just Android and Vita which use the full-screen only mode: 3DS, Wii, Wii U, Switch, etc.